packages/host/tests/acceptance/commands-test.gts

```diff
@@ -622,7 +623,7 @@ module('Acceptance | Commands tests', function (hooks) {
     assert.dom('[data-test-card-url-bar-input]').hasValue(`${testCard}.json`);
     await click('[data-test-submode-switcher] button');
     await click('[data-test-boxel-menu-item-text="Interact"]');
-    await click('[data-test-open-ai-assistant]');
```

Comment:
The behavior changed now that I put the `aiAssistantOpen` from the component to the state service–previously, if the AI panel was opened and the user toggled from interactive to code mode and then back to interactive, the AI panel was closed. But now, the AI panel remains open when going back to interactive mode. I think this behavior is actually more desirable, let me know if you think otherwise.